doc/man3/SSL_CONF_cmd.pod

```diff
@@ -171,6 +171,12 @@ Use server and not client preference order when determining which cipher suite,
 signature algorithm or elliptic curve to use for an incoming connection.
 Equivalent to B<SSL_OP_CIPHER_SERVER_PREFERENCE>. Only used by servers.
 
+=item B<-prioritize_chacha>
+
+Prioritize ChaCha ciphers when the client has a ChaCha20 cipher at the top of
+its preference list. This usually indicates a mobile client is in use.
```

Comment:
Mention that this option has no effect unless server preference is enabled. Replace "mobile" with client likely has no hardware AES support.  Perhaps mention that server performance may be impacted.

Reply:
The second statement implies x86_64 architecture, which is not always the case. I don't see any documentation referencing hardware AES support, and no mention of relative performance of cipher suites. i.e. the documentation doesn't specifically state anywhere that choosing ChaCha20 over AES in SSL_set_cipher() will impact server performance. But there's also a counter argument, that ChaCha may provide a boost in performance as far as the client is concerned. I don't think it's the place in the API documentation to go into over-arching performance issues that are better described in a whitepaper or something.